Makefile

```diff
@@ -541,7 +541,7 @@ nxOMSGenerateInventoryMof:
 
 nxOMSPlugin:
 	rm -rf output/staging; \
-	VERSION="3.57"; \
+	VERSION="3.58"; \
```

Comment:
Seth Rait just made changes to nxOMSPlugin and version is bumped to 3.58. Please work with him to ensure smooth rollout of combined changes